CoreFoundation/Collections.subproj/CFData.c

```diff
@@ -361,7 +361,10 @@ CF_SWIFT_EXPORT void _CFDataInit(CFMutableDataRef memory, CFOptionFlags flags, C
         }
         
         __CFDataSetNeedsToZero(memory, !cleared);
-        CFDataReplaceBytes(memory, CFRangeMake(0, 0), bytes, length);
+        // Avoid copying from null pointers.
+        if (length > 0) {
+            CFDataReplaceBytes(memory, CFRangeMake(0, 0), bytes, length);
```

Comment:
lets not change this... this is extraneous to the pull request and would require synchronization with CF.